components/DeleteButton.js

```diff
@@ -0,0 +1,23 @@
+import BinIcon from "@/public/binIcon.svg";
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+export default function DeleteButton({ toggleModalDeleteButton }) {
+  return (
+    <>
+      <Button onClick={() => toggleModalDeleteButton()}>
+        <StyledBinIcon />
+      </Button>
+    </>
+  );
+}
```

Comment:
Do you need to create a DeleteButton here? You could create a styled Button here that can be used in a variety of ways. Use the Icon where you use the general Button:
```js
export const Button = styled.button`
...
${({variant}) => variant === 'danger' && css`... styling for delete button`}
`
```